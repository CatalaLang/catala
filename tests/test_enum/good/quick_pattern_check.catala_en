## Article

```catala
declaration enumeration E:
  -- Case1 content integer
  -- Case2

declaration scope A:
  context x content E
  context y content boolean
  context z content boolean

scope A:
  definition x equals Case1 content 2
  definition y equals x with pattern Case1
  definition z equals x with pattern Case2
```
