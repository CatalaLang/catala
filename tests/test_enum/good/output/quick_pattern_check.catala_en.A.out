[RESULT] Computation successful! Results:
[RESULT] x = Case1 2
[RESULT] y = true
[RESULT] z = false
